genie-agent-app/build.gradle

```diff
@@ -25,6 +25,14 @@ license {
     exclude "*.txt"
 }
 
+dependencyManagement {
+    imports {
+        mavenBom "com.amazonaws:aws-java-sdk-bom:1.12.99"
+        mavenBom "io.awspring.cloud:spring-cloud-aws-dependencies:3.2.1"
```

Comment:
you should also 
* update Spring Boot version in https://github.com/Netflix/genie/blob/master/gradle.properties#L23-L24
* either reuse that `spring_cloud_version` var or remove it
* Spring Boot 3 integrates with flyway 10: https://github.com/Netflix/genie/blob/master/gradle.properties#L30, check if you can use Spring Boot BOM recommendation instead of hardcoded version